projects/hca_mvp/CreateAdapterMetadata.wdl

```diff
@@ -0,0 +1,206 @@
+version 1.0
+
+import "../../projects/tasks/CreateOptimusAdapterObjects.wdl" as CreateOptimusObjects
+import "../../projects/tasks/MergeOptimusLooms.wdl" as MergeLooms
+import "../../projects/tasks/AdapterTasks.wdl" as Tasks
+import "../../projects/tasks/CreateReferenceMetadata.wdl" as CreateReferenceMetadata
+
+
+workflow CreateAdapterMetadata {
+  meta {
+    description: "Creates json objects for indexing HCA analysis data"
+    allowNestedInputs: true
+  }
+
+  input {
+    Array[File] output_bams
+    Array[File] output_looms
+    Array[String] input_ids #sequencing_process_provenance_document_id
+    Array[String] fastq_1_uuids #array of space separated strings
+    Array[String] fastq_2_uuids #array of space separated strings
+    Array[String]? fastq_i1_uuids #array of space separated strings
+
+    # These values come in as arrays from Terra, but should be populated with a single value (which may be repeated)
+    Array[String] all_libraries
+    Array[String] all_organs
+    Array[String] all_species
+    Array[String] all_project_ids
+    Array[String] all_project_names
+
+    String output_basename
+    String cromwell_url = "https://api.firecloud.org/"
+    String staging_area = "gs://broad-dsp-monster-hca-prod-lantern/"
+    String version_timestamp = "2021-05-24T12:00:00.000000Z" # TODO should we hard code this?
```

Comment:
```suggestion
    String version_timestamp
```

Reply:
Ask if anyone has objections to this at DCP demos on 8/31. Timestamp should always be left at midnight (12:00:00.000000Z). Add this to spreadsheet/ask Marina to copy it in on SOP